oauth2_http/java/com/google/auth/oauth2/ExternalAccountCredentials.java

```diff
@@ -754,8 +745,12 @@ protected Builder(ExternalAccountCredentials credentials) {
       this.environmentProvider = credentials.environmentProvider;
       this.workforcePoolUserProject = credentials.workforcePoolUserProject;
       this.serviceAccountImpersonationOptions = credentials.serviceAccountImpersonationOptions;
-      this.universeDomain = credentials.universeDomain;
       this.metricsHandler = credentials.metricsHandler;
+      try {
```

Comment:
even if we don't remove the universeDomain field, can we have the builder copy the field without the try/catch/RTE -- what breaks?  I don't expect we test the RTE case.

Reply:
actually you are right... This could not be effectively used yet anyways, it was shipped as public by mistake. The rest of the universeDomain stuff in the credential was not public yet. It would be a zombie field in the builder until we remove it.  I will remove the try-catch and document that the field is not used.